Okta.Xamarin/Okta.Xamarin/OktaStateManager.cs

```diff
@@ -42,22 +47,24 @@ public class OktaState
         /// Gets the expiration time.
         /// </summary>
         public DateTime Expires { get; private set; }
-        
+
+        public IOktaConfig Config { get; set; }
+
         #region CTors
-        public OktaState()
+        public OktaStateManager()
         {
         }
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="OktaState"/> class.
+        /// Initializes a new instance of the <see cref="OktaStateManager"/> class.
         /// </summary>
         /// <param name="accessToken"></param>
         /// <param name="tokenType"></param>
         /// <param name="idToken"></param>
         /// <param name="refreshToken"></param>
         /// <param name="expiresIn"></param>
         /// <param name="scope"></param>
-        public OktaState(string accessToken, string tokenType, string idToken = null, string refreshToken = null, int? expiresIn = null, string scope = null)
+        public OktaStateManager(string accessToken, string tokenType, string idToken = null, string refreshToken = null, int? expiresIn = null, string scope = null)
```

Comment:
This constructor has too many params, I wonder if it'd make sense to create a convenience class with these attributes. What do you think?

Reply:
I tend to agree; most of the parameters are optional.  I may just remove the optional parameters and use object initialization instead.